validator/validator_spawner.go

```diff
@@ -0,0 +1,254 @@
+package validator
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/pkg/errors"
+)
+
+type ValidationSpawner struct {
+	machineLoader *NitroMachineLoader
+}
+
+func NewValidationSpawner(config NitroMachineConfig, fatalErrChan chan error) (*ValidationSpawner, error) {
+	// TODO
+	// // the machine will be lazily created if need be later otherwise
+	// if config.ArbitratorValidator {
+	// 	if err := validationSpawner.CreateMachine(validator.pendingWasmModuleRoot, true, false); err != nil {
```

Comment:
Is the plan to get rid of this code or re-enable it in a later PR?

Reply:
The plan is not to re-enable this code, but have some other way to preload the available machines